swap_meet/clothing.py

```diff
@@ -1,2 +1,12 @@
-class Clothing:
-    pass
\ No newline at end of file
+from .item import Item
+
+class Clothing(Item):
+    def __init__(self, id=None, condition=0, fabric="Unknown"):
+        super().__init__(id, condition)
+        self.fabric = fabric
+    
+    def get_category(self):
```

Comment:
For our children classes of `Item`,  we have a `get_category` method that essentially returns the name of the class. Though we are returning the different names (Clothing, Decor, Electronic) we could actually move this functionality into the parent class, `Item`. This is how it would look: 
```python
class Item:
...
def  get_category(self):
	return  self.__class__.__name__
```
`self` refers to the class instance that called it. Meaning if we have the following: 
```python
shirt = Clothing()

shirt.get_category() # returns "Clothing"
```
`self` in this case refers to `shirt`. The class of clothing is `Clothing`  and since `Clothing` is inheriting from the `Item` class it has access to the `get_category` method. This method is inherently being passed `shirt` and then accesses the `__class__` attribute and then the `__name__` attribute which gives us `Clothing`. Doing this will allow to make our code more DRY since we do this for `Clothing`, `Decor`, and `Electronic`.  This change could also allow us to make our `__str__` method more DRY as well.